classes/plugininfo.php

```diff
@@ -74,125 +71,53 @@ public static function get_plugin_configuration_for_context(
         ?\editor_tiny\editor $editor = null
     ): array {
 
-        $config = get_config('tiny_c4l');
-        $viewc4l = has_capability('tiny/c4l:viewplugin', $context);
-        $showpreview = get_config('tiny_c4l', 'enablepreview');
+        $config = get_config('tiny_elements');
+        $viewelements = has_capability('tiny/elements:viewplugin', $context);
+        $showpreview = get_config('tiny_elements', 'enablepreview');
         $isstudent = !has_capability('gradereport/grader:view', $context);
```

Comment:
uff...that's a bit ugly :) Can we move this to a tiny_elements specific capability?

Reply:
Is now tiny/elements:showteachercomponents